pkg/controller/controller.go

```diff
@@ -392,10 +392,26 @@ func (c *RolloutController) hasStatefulSetNotReadyPods(sts *v1.StatefulSet) (boo
 	// a terminating pod (which we don't consider "ready") may have not yet failed the
 	// readiness probe for the consecutive number of times required to switch its status
 	// to not-ready. For this reason, we list all StatefulSet pods and check them one-by-one.
```

Comment:
[nit] I would move the comment block ☝️ before the following line:

```
notReadyPods := notRunningAndReady(pods)
```

Reply:
Makes sense, moved in https://github.com/grafana/rollout-operator/pull/127/commits/85a9bb6227989a71dbf8e8d9ef5a87db32980cbf